net/net-snmp/Portfile

```diff
@@ -27,7 +27,9 @@ long_description        This is net-snmp, a derivative of CMU's SNMP \
                         handle SNMP traps, a version of the unix 'netstat' \
                         command using SNMP, a Tk/perl MIB browser.
 
-perl5.branches          5.28
+perl5.branches          5.28 5.30 5.32 5.34
+perl5.default_branch    5.32
```

Comment:
Why default to 5.32? 5.28 is the preferred default per https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/perl5-1.0.tcl#L17.